Documentation/O3DEMPSProjectExportTesting.md

```diff
@@ -32,13 +31,11 @@ scripts\o3de register --all-gems-path \path\to\workspace\o3de-multiplayersample-
 ```
 
 ## Run Project Export
-Use the MPSGamelift export script to produce a game and server package. 
+Use the O3DE Project-centric export script to produce a game and server package. 
 
 1. Create the directory you want to output the game and server, navigate to that directory, and run the export command:
 ```bash
-mkdir \path\to\output
-cd \path\to\output
-\full\path\to\workspace\o3de\scripts\o3de export-project -es \full\path\to\workspace\o3de-multiplayersample\MPSGameLift\Scripts\export_gamelift_server_package.py --code --assets -ll INFO --package-gamelauncher
+\path\to\workspace\o3de\scripts\o3de export-project -es \path\to\o3de\scripts\o3de\ExportScripts\export_standalone_monolithic_project_centric.py -pp \path\to\o3de-multiplayersample -out \full\path\to\output -cfg release -a zip -nounified -gpfp launch_client.cfg -spfp launch_client.cfg -code -assets -ll INFO -sl \path\to\o3de-multiplayersample\AssetBundling\SeedLists\BasePopcornFxSeedList.seed -sl \path\to\o3de-multiplayersample\AssetBundling\SeedLists\GameSeedList.seed -sl \path\to\o3de-multiplayersample\AssetBundling\SeedLists\VFXSeedList.seed 
```

Comment:
This code-block is really hard to read, and people typically like to copy/paste as much as they can. I'd suggestion to replace the  `\path\to\workspace\` with an an environment variable and have separate examples for windows and linux. 
You should add a blurb to set the variable `O3DE_PATH` to the path of the engine root ,`O3DE_PROJECT_PATH` to the path of the project root, and `OUTPUT_PATH` to the desired output path:

```suggestion

# Windows
%O3DE_PATH%\scripts\o3de export-project -es %O3DE_PATH%\scripts\o3de\ExportScripts\export_standalone_monolithic_project_centric.py -pp %O3DE_PROJECT_PATH% -out %OUTPUT_PATH% -cfg release -a zip -nounified -gpfp launch_client.cfg -spfp launch_client.cfg -code -assets -ll INFO -sl \path\to\o3de-multiplayersample\AssetBundling\SeedLists\BasePopcornFxSeedList.seed -sl %O3DE_PROJECT_PATH%\AssetBundling\SeedLists\GameSeedList.seed -sl %O3DE_PROJECT_PATH%\AssetBundling\SeedLists\VFXSeedList.seed 

#Linux
$O3DE_PATH/scripts/o3de export-project -es $O3DE_PATH/scripts/o3de/ExportScripts/export_standalone_monolithic_project_centric.py -pp $O3DE_PROJECT_PATH -out $OUTPUT_PATH -cfg release -a zip -nounified -gpfp launch_client.cfg -spfp launch_client.cfg -code -assets -ll INFO -sl $O3DE_PROJECT_PATH/AssetBundling/SeedLists/BasePopcornFxSeedList.seed -sl $O3DE_PROJECT_PATH/AssetBundling/SeedLists/GameSeedList.seed -sl $O3DE_PROJECT_PATH/AssetBundling/SeedLists/VFXSeedList.seed

```